lib/srv/uacc/uacc.h

```diff
@@ -78,6 +92,8 @@ static int check_abs_path_err(const char* buffer) {
 
     // check for GNU extension errors
     if (errno == EACCES || errno == ENOENT) {
+        UACC_PATH_ERR = (char*)malloc(PATH_MAX);
```

Comment:
I'm not C POSIX export but does the `buff != NULL &&(errno == EACCES || errno == ENOENT)` can really happen ?
The input parameter `const char* buffer` in current flow is always result of the`realpath(path, buffer);` call where `realpath` RETURN VALUE  is defined as: [realpath](https://man7.org/linux/man-pages/man3/realpath.3.html#top_of_page)

> If there is no error, realpath() returns a pointer to the resolved_path.
> 
> Otherwise, it returns NULL, the contents of the array
resolved_path are undefined, and [errno](https://man7.org/linux/man-pages/man3/errno.3.html) is set to indicate the
error.

So based on POSIX [errno doc ](https://man7.org/linux/man-pages/man3/errno.3.html) if errno != 0 the `buffer` == NULL

Reply:
This isn’t posix standardized, this is a glibc specific extension that’s sparsely documented . POSIX is a nice standard but a lot of stuff deviates and has subtle differences from it by now. This is one of them.

Reply:
This was mentioned in the top level description of the PR:
`This PR also adds additional diagnostics if realpath decides to fail in such a way that glibc exposes extra debugging info as a non-posix extension`

Reply:
Added comment to the source for any future reader.

Reply:
All right, Thanks for the explanation. Then it LGTM.